plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/IcebergOptimizeHandle.java

```diff
@@ -97,4 +97,15 @@ public boolean isRetriesEnabled()
     {
         return retriesEnabled;
     }
+
+    @Override
+    public String toString()
+    {
+        return new StringBuilder()
+                .append("procedure").append(":").append("optimize")
+                .append("tableColumns").append(":").append(tableColumns)
+                .append("fileFormat").append(":").append(fileFormat)
+                .append("maxScannedFileSize").append(":").append(maxScannedFileSize)
+                .toString();
```

Comment:
in the meantime `retriesEnabled` was added to the class, add it to the toString too